### ISSUE TITLE
Text position and size different for SVG vs Rasterized

### ISSUE THREAD

**Issue by notflip** [meyfa/php-svg#111]
This is the output from the SVG xml
<img width="303" alt="image" src="https://user-images.githubusercontent.com/4146384/98434415-9086bb00-20cf-11eb-8450-3f451470cd7c.png">

This is what it looks like when using the rasterize method.
<img width="81" alt="image" src="https://user-images.githubusercontent.com/4146384/98434420-98465f80-20cf-11eb-97bb-8bd9ea3219d6.png">

This is my code

```
        $image = new SVG(280, 280);
        $doc = $image->getDocument();

        $background = new SVGRect(0, 0, 280, 280);
        $background->setStyle('fill', '#ff4444');
        $doc->addChild($background);

        $text = (new SVGText($initials, '50%', '50%'))
            ->setFont(new SVGFont('Raleway', 'fonts/raleway.ttf'))
            ->setSize(120);

        $text->setStyle('line-height', 1);
        $text->setAttribute('text-anchor', 'middle');
        $text->setAttribute('alignment-baseline', 'middle');

        $doc->addChild($text);
```

**Comment by meyfa** [comment]
`text-anchor` added via #97. `alignment-baseline` still pending.

**Comment by bogart0324** [comment]
hi, beside from this setFont(new SVGFont('Raleway', 'fonts/raleway.ttf')), what else need to do to make the font working?  I tried it but seem the correct font is not applied.

**Comment by notflip** [comment]
@bogart0324 It's probably in the wrong folder? My fonts folder is located in my public  folder, from which my php script (index.php) is being ran.

**Comment by bogart0324** [comment]
yes same with me, I manage to make it work by replacing sprintf to printf
in SVGFont class constructor, see image attached, I hope I din't mess it up
:)
[image: image.png]
